CRM/Upgrade/Incremental/php/FiveSixtySeven.php

```diff
@@ -36,6 +36,53 @@ public function upgrade_5_67_alpha1($rev): void {
     $this->addTask('Add cache_fill_took column to Group table', 'addColumn', 'civicrm_group', 'cache_fill_took',
       'DOUBLE DEFAULT NULL COMMENT "Seconds taken to fill smart group cache, not always related to cache_date"',
       FALSE);
+    $this->addTask('Update civicrm_mailing_event_queue to permit deleting records from civicrm_mailing_job', 'updateMailingEventQueueTable', 'updateMailingEventQueueTable');
```

Comment:
@eileenmcnaughton this upgrade step has [reportedly caused someone's upgrader to crash](https://civicrm.stackexchange.com/questions/46045/error-executing-task-update-civicrm-mailing-event-queue-to-permit-deleting-reco). Any idea why?

Reply:
@colemanw table size I expect